packages/react/src/rsc/streamable-ui/create-streamable-ui.tsx

```diff
@@ -1,4 +1,4 @@
-import { HANGING_STREAM_WARNING_TIME_MS } from '../../util/constants';
+import { HANGING_STREAM_WARNING_TIME_MS } from 'ai';
```

Comment:
I see - let me think about it. We may as well duplicate the value and remove the constant.

Reply:
(since I'll clean up the `ai` parts that use it most likely in v5)

Reply:
fixed